tests/unit/llms/test_llm_loader.py

```diff
@@ -90,3 +90,15 @@ def test_load_llm(_registered_fake_provider):
 
     llm = load_llm(provider="fake-provider", model="model")
     assert llm == "fake_llm"
+
+
+@patch("ols.constants.SUPPORTED_PROVIDER_TYPES", new=["fake-provider"])
```

Comment:
@tisnik i'm not seeing why you needed to add a fake provider.  The load_llm coded doesn't check if it's a known provider type until after it has thrown the "provider config missing" exception:

https://github.com/openshift/lightspeed-service/blob/c52740fbcf0d9115a6bcfffa3387f81ab50511f5/ols/src/llms/llm_loader.py#L79-L87

nor is anything else being checked, so you can pass whatever provider name you want, valid or not, it won't affect the test behavior.

Reply:
that's actually right. Previously the code was different and after some refactoring, the mock is not needed. Sharp :eye:

Reply:
i look for any opportunity i can to remove mocks from our code :)